recipes/nrpys/meta.yaml

```diff
@@ -7,8 +7,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 3
-  noarch: python
```

Comment:
Why the recipe is arch specific now ?
Being 'noarch' means it could be used on any CPU architecture